cmd/cluster/create_cluster.go

```diff
@@ -209,7 +209,7 @@ func init() {
 	createClusterCmd.Flags().String("cloud-provider", "", "[OPTIONAL] The cloud provider where database needs to be deployed. AWS, AZURE or GCP. Default AWS.")
 	createClusterCmd.Flags().String("cluster-tier", "", "[OPTIONAL] The tier of the cluster. Sandbox or Dedicated. Default Sandbox.")
 	createClusterCmd.Flags().String("cluster-type", "", "[OPTIONAL] Cluster replication type. SYNCHRONOUS or GEO_PARTITIONED. Default SYNCHRONOUS.")
-	createClusterCmd.Flags().String("database-version", "", "[OPTIONAL] The database version of the cluster. Production, Innovation or Preview. Default depends on cluster tier, Sandbox is Preview, Dedicated is Production.")
+	createClusterCmd.Flags().String("database-version", "", "[OPTIONAL] The database version of the cluster. Production, Innovation, Preview or 'Early Access'. Default depends on cluster tier, Sandbox is Preview, Dedicated is Production.")
```

Comment:
```suggestion
	createClusterCmd.Flags().String("database-version", "", "[OPTIONAL] The database version of the cluster. Production, Innovation, Preview, or 'Early Access'. Default depends on cluster tier, Sandbox is Preview, Dedicated is Production.")
```

Reply:
Request to remove:
 --host string YBM Api hostname
Why should we externalize this option since it is primarily an internal option for YB only?

Reply:
You are correct. Removed it. Thanks for identifying this.